help/versions.md

```diff
@@ -1,59 +1,95 @@
-the above example, `https://arxiv.org/abs/1710.02185`. For information
-on how to cite individual versions, please see the related
-[references](faq/references) FAQ page.
-
-What constitutes a version?
----------------------------
+# Submission Version Availability
+
+arXiv is intended as a historical collection of research works. Once made public, each version of a work is considered a permanent part of the scientific record and may not be removed.
+
+If an author needs to update the article after announcement, they may replace or withdraw the article, which creates a new version. This increments the version portion of the arXiv identifier. An example article with multiple versions is [arXiv:1710.02185](https://arxiv.org/abs/1710.02185).
+
+Making previous versions of submitted papers publicly available archives the historical record of research work. As such, it is a useful aid to peer review and verification of intellectual priority.
+
+
+## What does arXiv mean by versions?
+
+
+All submissions receive a "date stamp", recorded as the time the submitter clicks "Submit article" at the end of the submission process. Once a submission is made available, it receives a permanent  [arXiv identifier](arxiv_identifier). Any replacement or withdrawal of that article generates a new version. This will increment its version number by one, with the most recent version always being displayed, even when the Abstract page is viewed without a version number at the end, such as the above example, https://arxiv.org/abs/1710.02185. 
+
+
+## Citing versions
+
+Authors can cite each version with the full identifier that includes the version number. arXiv:YYMM.NNNNNvX
+
+Authors can also use the Comments metadata to describe the different versions.
+
+Ex:
+> For the longer conference version see: arXiv:YYMM.NNNNNvv1. For the shortened summary see arXiv:YYMM.NNNNNvv2, for the journal version that includes extended data in the appendix see arXiv:YYMM.NNNNNv3
+
+Please see the [references and citations help page](./faq/references) for details on citing works in arXiv.
+
+
+## What constitutes a version?
 
 A version will be created once a [replacement](replace) or
-[withdrawal](withdraw) submission is made public. The *Submission
-history* section of the Abstract page indicates the version number being
-viewed, as well as all available versions.
+[withdrawal](withdraw) submission is made public. The _Submission history_ section of the Abstract page indicates the version number being viewed, as well as all available versions.
+
+Replacements will contain the full text of the article. Withdrawals will display the reason for withdrawal entered as a comment, without any content download options for that version.
+
+
+## Situations when arXiv typically requires versioning
+
+Below are cases where arXiv authors are asked to version their work under a single arXiv identifier rather than to submit related works as new arXiv identifier.

```

Comment:
```suggestion
Below are cases where arXiv authors are asked to version their work under a single arXiv identifier rather than submit related works as new arXiv identifier.

```